tests/platforms/macOS/app/test_run.py

```diff
@@ -298,12 +309,89 @@ def test_run_gui_app_test_mode(
     run_command.tools.os.kill.assert_called_with(100, SIGTERM)
 
 
+def test_run_gui_app_debugger(
+    run_command, first_app_config, sleep_zero, tmp_path, monkeypatch, dummy_debugger
```

Comment:
Style nit - as with arguments in function calls, if the argument list is long enough to split like this, we prefer to go straight to "1 argument per line" form, rather than this form.